### PR TITLE
feat: add throttling to unified slider

### PR DESCRIPTION
**Pull request by mariobuikhuizen**

### CODE REVIEW

File: jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class UnifiedSlider(TemplateMixin):
     min_value = Float(0).tag(sync=True)
     max_value = Float(100).tag(sync=True)
     linked = Bool(True).tag(sync=True)
+    wait = Int(300).tag(sync=True)
```
Comment (rosteen) [inline jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py:20]: This is great and is working well, but I think 100 ms here for the throttle value strikes a better balance for responsiveness. @dtn5ah @javerbukh would one of you give this a try and see if you agree?

```suggestion
    wait = Int(100).tag(sync=True)
```

**Review by rosteen — CHANGES_REQUESTED** [review]

File: jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py (hunk 1 above)
Comment (javerbukh) [reply jdaviz/configs/cubeviz/plugins/unified_slider/unified_slider.py:20]: Yes, 100 feels much smoother.

**Review by javerbukh — APPROVED** [review]
I agree with Rickys comment about using 100 instead of 300, but otherwise this looks and runs great!

**Comment by rosteen** [comment]
Closes https://github.com/spacetelescope/jdaviz/issues/244

**Pull request merged**